pages/common/lwp-request.md

```diff
@@ -0,0 +1,27 @@
+# lwp-request
+
+> Simple Perl-based command-line HTTP client.
```

Comment:
We never mentioned what language program is written. It should not matter for newbie.